### ISSUE TITLE
Microsoft.Azure.WebJobs.Extensions.Storage  Could not load type 'Microsoft.Azure.WebJobs.Host.Scale.IScaleMonitorProvider'

### ISSUE THREAD

**Issue by pastushenkoy** [Azure/azure-webjobs-sdk-extensions#628]
Please provide a succinct description of the issue.

Version `3.0.10` of package `Microsoft.Azure.WebJobs.Extensions.Storage`
breaks function runtime.

#### Repro steps

I'm using functions in `mcr.microsoft.com/azure-functions/dotnet:2.0-alpine` container. It worked fine until I updated `Microsoft.Azure.WebJobs.Extensions.Storage` to version `3.0.10`. I checked previous versions of this library and it worked in every version until `3.0.10` including `3.0.8` 

#### Expected behavior

Extected working functions

#### Actual behavior

When function starts I'm getting error:
```
The function runtime is unable to start. Microsoft.Azure.WebJobs.Extensions.Storage: Could not load type 'Microsoft.Azure.WebJobs.Host.Scale.IScaleMonitorProvider' from assembly 'Microsoft.Azure.WebJobs.Host, Version=3.0.12.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.
Session Id: 5ddf690d4b294ae488bd802248665de8

Timestamp: 2020-02-19T13:40:44.629Z
```

#### Known workarounds

Use version `3.0.8`

**Comment by fabiocav** [comment]
@ahmelsayed is that container image stale? Is that the one recommended? The exception seems to indicate it is lagging behind and doesn't have the newer SDK references.

**Comment by ahmelsayed** [comment]
yes that alpine image is out of date. Use `mcr.microsoft.com/azure-functions/dotnet:2.0` instead.
